src/hooks/useIntegratedNotifications.ts

```
@@ -0,0 +1,94 @@
+  enableFCM = true,
+  enableSSE = true,
+  pageSize = 20,
+}: UseIntegratedNotificationsProps) => {
+  // 새로운 통합 서비스를 우선 사용 (FCM + SSE 통합)
+  const useIntegratedService = enableSSE && enableFCM;
+
+  // 새로운 통합 알림 서비스 (FCM + SSE)
+  const integratedNotification = useIntegratedService ? useNotification({
+    userId,
+    autoConnect: enableSSE,
+    enableBrowserNotifications: enableFCM
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)